src/components/AvatarList/AvatarList.jsx

```diff
@@ -50,7 +55,7 @@ const getCurrentCount = ({ count, max }) => (count < max ? count : max)
 export const getAvatarSize = ({ size: sizeProp, stack, ...rest }) => {
   const currentCount = getCurrentCount(rest)
 
-  if (!stack) return sizeProp
+  if (stack !== 'horizontal') return sizeProp
```

Comment:
@tinkertrain could we add support for consuming the `sizeProp` no matter the stack setting?